CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+
+## [0.9.4-beta] - 2016-04-20
+### Added
+- Deploy to App Engine flexible environment tools menu item. ([#635](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/635))
+- Support for Maven based projects as deployment sources for App Engine flexible environment deployments ([#600](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/600))
+
+### Changed
+- App Engine flexible environment deployment can be cancelled by disconnecting to our App Engine applicaiton server. ([#581](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/581))
```

Comment:
Yeah adding instructions to the contributors is a good idea! I'll do that
as part of this change.

On Mon, May 2, 2016 at 11:06 AM, Etan Shaul <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/668#discussion_r61751178>
> :
>
> > @@ -0,0 +1,18 @@
> > +# Change Log
> > +All notable changes to this project will be documented in this file.
> > +
> > +## [Unreleased]
> > +
> > +
> > +## [0.9.4-beta] - 2016-04-20
> > +### Added
> > +- Deploy to App Engine flexible environment tools menu item. ([#635](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/635))
> > +- Support for Maven based projects as deployment sources for App Engine flexible environment deployments ([#600](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/600))
> > +
> > +### Changed
> > +- App Engine flexible environment deployment can be cancelled by disconnecting to our App Engine applicaiton server. ([#581](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/581))
>
> Also noting that if we leave it up to contributors to do this it would be
> nice to add the instruction in our project contributors file too.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/668/files/04b02fffca09c4a43dc7250df90cbeebcea29395#r61751178>
>